README.md

```diff
@@ -149,6 +149,15 @@ Claim or reclaim name `test2` with password `secret123`:
 Unclaim name `test2` using password `secret123`:
 > [armoria.herokuapp.com/unclaim/test2/secret123](https://armoria.herokuapp.com/unclaim/test2/secret123)
 
+## Local devlopment
+
+Make sure you have the following packages installed
+
+```bash
+brew install pkg-config cairo pango libpng jpeg giflib
```

Comment:
Are you sure these packages are required?

Reply:
I needed them in order to install the packages. I tried many suggestions online and this what helped. I was getting errors related to `node-pre-gyp` and `canvas`.
I found this suggestion in some question. I see they are mentioned in here https://github.com/Automattic/node-canvas/blob/master/Readme.md

Reply:
Need to cross-check. If they are required, they should be mentioned in `package.json` dependencies

Reply:
Sure